cytopipe/cli/exec/workflow_exec.py

```diff
@@ -17,6 +23,14 @@ def exec_preprocessing(n_cores=1) -> int:
         list of par
     n_cores : int, optional
         max number of cores to use in the workflow, by default 1
+    use_conda_env : bool, optional
```

Comment:
why would this ever be false? just curious

Reply:
I guess here I was thinking about giving the user "more freedom". meaning that are not always bound to our environment presets. They can use their own environments if they can.

As of writing this, now I can see how nasty this can get, in terms of reproducibility. 

This parameter has been removed on commit: f80f6fc5d3e1e4ed71bf1c1f5dc28d76e8717b00